src/lib/lib3index/cholesky.cc

```diff
@@ -1,12 +1,7 @@
 /*
- * @BEGIN LICENSE
+ *@BEGIN LICENSE
```

Comment:
What is the best way of doing this?  I tried updating my master branch which is completely updated.  I thought that the pull request will do the merge.